Improve API 4.0 documentation (#2812)
wazuh/wazuh-documentation: source/user-manual/api/index.rst

RESTful API
===========

> The Wazuh API is an open source RESTful API that allows for interaction with the Wazuh manager from a web browser, command line tool like cURL or any script or program that can make web requests. The Wazuh UI relies on this heavily and Wazuh's goal is to accommodate complete remote management of the Wazuh infrastructure via the Wazuh Kibana app. Use the API to easily perform everyday actions like adding an agent, restarting the manager(s) or agent(s) or looking up syscheck details.

Reviewer: ```suggestion
The Wazuh API is an open source RESTful API that allows for interaction with the Wazuh manager from a web browser, command line tool like cURL or any script or program able to make web requests. The Wazuh UI relies on the Wazuh API and Wazuh's ultimate goal is to accommodate complete remote management of the Wazuh infrastructure via the Wazuh UI. Use the Wazuh API to easily perform everyday actions like adding an agent, restarting the manager(s) or agent(s) or looking up syscheck details.
```
